src/Pages/Cart/Cart.js

```diff
@@ -1,9 +1,16 @@
 import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
 
 class Cart extends Component {
+  componentDidMount() {
+    fetch(`http://10.58.6.70:8000/productdetails/${this.props.match.params.id}`)
```

Comment:
```suggestion
    fetch(`${API}/productdetails/${this.props.match.params.id}`)
```
- src/config.js에 api 주소 모아서 관리해주시면 관리하기 편합니다!